.github/workflows/ett.yml

```diff
@@ -117,119 +117,52 @@ jobs:
           path: ./dist/apps/ett/${{ env.app-name }}/
           if-no-files-found: error
 
-  container-pr:
-    name: 'PR: Host container'
-    if: ${{ github.event_name  == 'pull_request' }}
-    needs: app
+  define_build:
+    name: Define build matrix
     runs-on: ubuntu-latest
-    env:
-      host-image-tag: ${{ format('pr-{0}', github.event.pull_request.number) }}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/download-artifact@v2
-        with:
-          name: ${{ env.app-name }}
-          path: ./dist/apps/ett/${{ env.app-name }}/
-      - name: Container image name
-        id: image-name
-        uses: ASzc/change-string-case-action@v2
-        with:
-          # TODO(xlgni): coordinate name change to `env.app-name` with Fenris infrastructure
-          string: ghcr.io/${{ github.repository }}-ett-app
-      - name: Build host container image
-        run: docker build ./ --file ./infrastructure/eo/host/Dockerfile --tag ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
-      - name: Publish host container image
-        run: |
-          docker login ghcr.io --username ${{ github.actor }} --password ${{ github.token }}
-          docker push ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
-
-  container_merge:
-    name: 'Merge: Host container'
-    if: ${{ github.ref == 'refs/heads/main' }}
-    needs: app
     outputs:
-      release_tag: ${{ steps.build_and_push.outputs.RELEASE_VERSION }}
-    runs-on: ubuntu-latest
+      chart_version: ${{ steps.get_semver.outputs.semver }}
+      containers: >-
+        [
+          { 
+            "deployment_name": "app", 
+            "dockerfile": "infrastructure/eo/host/Dockerfile", 
```

Comment:
suggestion: If we have time, we would like to change the path `infrastructure/eo/host` to `build/infrastructure/eo/host` but then I think we need to change a configuration setting in `eo-base-environment` as well.

Reply:
Once it has moved to the new workflow, this dependency is removed from the base-environment, and then this can be easily changed locally.